pkg/controllers/monitoring/monitoring-stack/conditions.go

```diff
@@ -0,0 +1,121 @@
+package monitoringstack
+
+import (
+	monv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	stack "github.com/rhobs/observability-operator/pkg/apis/monitoring/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	monitoringStackAvailable       = "MonitoringStackAvailable"
+	monitoringStackReconciled      = "MonitoringStackReconciled"
+	prometheusNotAvailable         = "PrometheusNotAvailable"
+	prometheusNotReconciled        = "PrometheusNotReconciled"
+	cannotReadPrometheusConditions = "Cannot read Prometheus status conditions"
+)
+
+type conditions map[stack.ConditionType]*stack.Condition
```

Comment:
To avoid any issue with nil pointers
```suggestion
type conditions map[stack.ConditionType]stack.Condition
```

Reply:
Updated. Thanks

Reply:
Well why? I think it's easier & better to work with pointers.

Reply:
you don't have to check for nil pointers so no risk of panics